js/huobipro.js

```diff
@@ -145,6 +145,9 @@ module.exports = class huobipro extends Exchange {
                 'api-signature-not-valid': AuthenticationError, // {"status":"error","err-code":"api-signature-not-valid","err-msg":"Signature not valid: Incorrect Access key [Access key错误]","data":null}
                 'base-record-invalid': OrderNotFound, // https://github.com/ccxt/ccxt/issues/5750
             },
+            'exceptionMsgs': {
+                'invalid symbol': BadSymbol,
```

Comment:
@brandsimon is there an `err-code` corresponding to this error?

Reply:
@kroitor There is an err-code, but it just says invalid order or invalid parameter. It is not specific for the symbol.

Reply:
@brandsimon got it... I suggest that we place that under `exceptions` in `describe()`, splitting the exceptions to `exact` and `broad`, as we do with many other exchanges that have two types of recognized errors: codes and messages. 

For example:

- https://github.com/ccxt/ccxt/blob/master/js/bitfinex.js#L312

The problem with `exceptionMsgs` is that the property does not exist in the base class in Py/PHP. It works in JS, because JS allows assigning instance properties in runtime. But when you access it with Python or PHP it will throw an exception there. Therefore all props have to be predefined, and in that case it's easier to add new errors to the existing `exceptions` property, rather than add a new base property to each lang.

Let me know if you can modify it slightly to account for the above.

Reply:
@kroitor I tested it with python and php.
```
 $ .venv/bin/python examples/py/cli.py bittrex fetch_ohlcv SOME/RANDOM
Traceback (most recent call last):
  File "examples/py/cli.py", line 126, in <module>
    result = method(*args)
  File "/home/work/ccxt/python/ccxt/bittrex.py", line 580, in fetch_ohlcv
    market = self.market(symbol)
  File "/home/work/ccxt/python/ccxt/base/exchange.py", line 1677, in market
    raise BadSymbol('{} does not have market symbol {}'.format(self.id, symbol))
ccxt.base.errors.BadSymbol: bittrex does not have market symbol SOME/RANDOM

 $ php examples/php/cli.php bittrex fetch_ohlcv SOME/RANDOM
bittrex->fetch_ohlcv (SOME/RANDOM)
ccxt\BadSymbol: bittrex does not have market symbol SOME/RANDOM
```

Reply:
@kroitor 
I can change it anyway, but the I'd rather also match the msg exact as the code.
So would it be ok to use
```
'exceptions': {
    'err-code': {...},
    'err-msg': {..},
}
```

Reply:
@kroitor 
I thought I tested it with python/php and it worked.
```
 $ php examples/php/cli.php huobipro fetch_ohlcv CDC/ETH
huobipro->fetch_ohlcv (CDC/ETH)
ccxt\BadSymbol: huobipro {"ts":1568813334794,"status":"error","err-code":"invalid-parameter","err-msg":"invalid symbol"}

 $ .venv/bin/python examples/py/cli.py huobipro fetch_ohlcv CDC/ETH
Traceback (most recent call last):
  File "examples/py/cli.py", line 126, in <module>
    result = method(*args)
  File "/home/work/ccxt/python/ccxt/huobipro.py", line 493, in fetch_ohlcv
    response = self.marketGetHistoryKline(self.extend(request, params))
  File "/home/work/ccxt/python/ccxt/base/exchange.py", line 436, in inner
    return entry(_self, **inner_kwargs)
  File "/home/work/ccxt/python/ccxt/base/exchange.py", line 459, in request
    return self.fetch2(path, api, method, params, headers, body)
  File "/home/work/ccxt/python/ccxt/base/exchange.py", line 455, in fetch2
    return self.fetch(request['url'], request['method'], request['headers'], request['body'])
  File "/home/work/ccxt/python/ccxt/base/exchange.py", line 563, in fetch
    self.handle_errors(http_status_code, http_status_text, url, method, headers, http_response, json_response, request_headers, request_body)
  File "/home/work/ccxt/python/ccxt/huobipro.py", line 957, in handle_errors
    raise exceptionMsgs[msg](feedback)
ccxt.base.errors.BadSymbol: huobipro {"ts":1568813361764,"status":"error","err-code":"invalid-parameter","err-msg":"invalid symbol"}
```

I will adapt the PR anyway.